buildspec-py3.yml

```diff
@@ -193,7 +193,15 @@ phases:
 
 
 reports:
-  reportGroupJunit:
+  tests:
     files:
       - "junit-reports/**/*"
     file-format: JunitXml
+  cov_func:
+    files:
+      - "coverage_functional.xml"
+    file-format: COBERTURAXML
+  cov_int:
+    files:
+      - "coverage_integration.xml"
+    file-format: COBERTURAXML
```

Comment:
Here I would only use one group for coverage in Codebuild. Otherwise we will end up have hundreds of groups.

```yaml
reports:
  cover:
    files:
      - "coverage*.xml"
    file-format: COBERTURAXML
```

Reply:
If having multiple files for coverage in Codebuild is an issue, then I'll drop out the coverage report in codebuild.